### ISSUE TITLE
Error SQLSTATE[42S02]: Base table or view not found on install

### ISSUE THREAD

**Issue by luizeof** [saelos/saelos#5]
When running installer I got this error on /install/final path

string(420) "SQLSTATE[42S02]: Base table or view not found: 1146 Table 'saelos_powertic_com.oauth_clients' doesn't exist (SQL: insert into `oauth_clients` (`user_id`, `name`, `secret`, `redirect`, `personal_access_client`, `password_client`, `revoked`, `updated_at`, `created_at`) values (, Saelos Personal Access Client, pepmpY2oQUgeneLeWaEHKVOR082pVsDBd27fiyhR, http://localhost, 1, 0, 0, 2018-04-16 08:04:34, 2018-04-16 08:04:34))"

**Comment by denisdulici** [comment]
I got the same error:

![2018-04-16_1514](https://user-images.githubusercontent.com/5254835/38808512-2ae80ad8-4189-11e8-8547-678ce57a16bd.png)

**Comment by dbhurley** [comment]
Looking into this now.

**Comment by Arenhardt** [comment]
+1 here. Same problem

**Comment by dongilbert** [comment]
Try running the following commands:

`php artisan migrate --force`
`php artisan passport:install --force`
`php artisan db:seed`

**Comment by AlanRezende** [comment]
When I Try `php artisan migrate --force`
```
In Connection.php line 664:

  SQLSTATE[42S02]: Base table or view not found: 1146 Table 'labsturbinabiz_s
  aelos.roles' doesn't exist (SQL: select * from `roles`)


In PDOConnection.php line 79:

  SQLSTATE[42S02]: Base table or view not found: 1146 Table 'labsturbinabiz_s
  aelos.roles' doesn't exist


In PDOConnection.php line 77:

  SQLSTATE[42S02]: Base table or view not found: 1146 Table 'labsturbinabiz_s
  aelos.roles' doesn't exist
```

**Comment by RabiCodie** [comment]
i am getting only blank page after installer is done, i have cloned the repository and run composer install however still problems rendering pages.

**Comment by dongilbert** [comment]
@RabiCodie After cloning and doing a composer install, you'll need to run `npm install && npm run prod` in order to get the app working.

@AlanRezende It seems there was an issue during installation. I would recommend starting over with 1.0-beta-3 as some installation issues were addressed in that release.

@luizeof Please try Beta 3 to see if it addresses your install issues. Please open a new issue if the problem persists afterwards.

**Comment by dongilbert** [comment]
@moazam1 Thank you for reporting the `active` issue. I've fixed it here: 92c4dd3
